features/keychain/api/index.d.ts

```diff
@@ -4,6 +4,17 @@ import BN from 'bn.js'
 type SeedId = string
 type KeySource = { seedId: SeedId; keyId: KeyIdentifier }
 
+type Tx = Record<string, unknown>
```

Comment:
Do we already have this defined somewhere?

Reply:
https://github.com/ExodusMovement/exodus-hydra/blob/36b4edd68d5e99993194375f1c685fe6228b2b28/features/tx-signer/src/module/interfaces.ts#L31

Reply:
Thanks I used the ones I found relevant in my recent PR